### ISSUE TITLE
Text alignment has no effect on multirow legend text

### ISSUE THREAD

**Issue by ftvalentini** [has2k1/plotnine#881]
For example:

```python
import pandas as pd
from plotnine import ggplot, aes, geom_point, scale_color_manual, theme, element_text
from sklearn.datasets import load_iris

iris = load_iris(as_frame=True)
df = iris.frame

# Modify species labels to include line breaks for wrapping
long_labels_wrapped = {
    0: 'Setosa - a long\ndescriptive name',
    1: 'Versicolor - another\nlong description',
    2: 'Virginica - yet another\nlong description'
}
df['species'] = df['target'].map(long_labels_wrapped)

plot = (
    ggplot(df, aes('sepal length (cm)', 'sepal width (cm)', color='species'))
    + geom_point()
    + theme(
        legend_text_legend=element_text(ha='left'), 
    )
)

print(plot)
```

displays centered color labels.

![image](https://github.com/user-attachments/assets/3be8ae57-8ab4-4a2e-8d84-11acc1efaee2)

In fact using any value for `ha` or `hjust` yields centered labels.

Using:
python 3.11.9 (main, Apr 19 2024, 16:48:06) [GCC 11.2.0]
plotnine 0.13.6

**Comment by has2k1** [comment]
Use `ma="left"`.

**Comment by nathancunn** [comment]
> Use `ma="left"`.

This similarly has no effect in plotnine 0.14.3 using the above reproducible example.
